src/scala/com/github/johnynek/bazel_deps/IO.scala

```diff
@@ -16,18 +17,24 @@ import cats.implicits._
  */
 
 object IO {
+  val charset = "UTF-8"
+  val pathSeparator = "/"
```

Comment:
should we use `File.separator` here?

Reply:
Probably?  I wasn't sure if there was a reason you were using the less-portable explicit `/`.

Reply:
updated